components/dropdown/dropdown-menu.js

```diff
@@ -12,20 +12,29 @@ import { ThemeMixin } from '../../mixins/theme-mixin.js';
  */
 class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 
+	static get properties() {
+		return {
+			_noAutoFocus: { type: Boolean }
+		};
+	}
+
 	static get styles() {
 		return dropdownContentStyles;
 	}
 
 	constructor() {
 		super();
-		this.noAutoFocus = true;
 		this.noPadding = true;
 		this._maxHeightNonTray = this.maxHeight;
+		this._noAutoFocus = false;
```

Comment:
Yup, I think that could also work. 
I am also just overall a bit concerned about the fact that there's this property that in theory is on `d2l-dropdown-menu` but doesn't actually do anything, so if we were to do that then I think we'd also want to somehow document the fact that `noAutoFocus` doesn't actually work with `d2l-dropdown-menu` (and also `noPadding`).

Reply:
So I gave this a try and it technically can work, however it doesn't really work for our case in Daylight because we're using `toggleOpen()` within a helper script in order to not have the `opened` appear in the code, since we wouldn't want that copy/pasted into someone else's code. Because of this, that would really complicate the logic since it's not going to be opened on first render, but slightly later.